cumulus_library/databases.py

```diff
@@ -497,3 +499,47 @@ def create_db_backend(args: dict[str, str]) -> DatabaseBackend:
         raise ValueError(f"Unexpected --db-type value '{db_type}'")
 
     return backend
+
+
+def upload_file(
+    *,
+    cursor: DatabaseCursor,
+    file: pathlib.Path,
+    study: str,
+    topic: str,
```

Comment:
Will `topic` always be a sensible thing to provide? Maybe allow this to be null or just skip it and prefix the filename for the `vocab` study? But it's also fine to leave here - just felt like a level of organization not all studies will need.